content/05.conclusions.md

```diff
@@ -3,4 +3,53 @@
+We also found, though, that for some paragraph the revision was not good enough, and in some cases it even made the text worse.
+This was particularly true for short paragraphs and some in the the Methods sections when mathematical expressions were present.
+
+Abstracts:
+
+  - remember that the models are stochastic (meaning that we can always run it again).
+  - for the phenoplier abstract, the revision was not really great (the AI model removed too much background information).
+    - the user of the tool, in these cases, might want to run it again to obtain another revision.
+    - for abstract, maybe we need to improve the prompts to say something like "keep more background information".
+
+
+Introductions:
+
+  - really good for both manuscripts
+  - in phenoplier, the models did not keep the format of citations for one paragraph, although this can be easily fixed by a human.
+    - the other paragraph that did not converge was a large one; this could be an indication that is not easy to understand and it should be split into smaller paragraphs.
+    - therefore, the fact that the tool cannot process a paragraph of text can also provide insight about the quality of the writing.
+
+
+Methods:
+  - the model not only improved the text, but also fixed references to wrong mathematical symbols.
+
+
+One way to fix the problem of context is to use chatbots like ChatGPT, which could enable the same paragraph-by-paragraph approach to revision that we presented here, but with higher quality revisions by considering the context (just as if it were reading it).
+  - This approach (chatGPT, etc) has also the advantage of being independent of the context length of the models.
+  - Here, mention this kind of models in general again, besides ChatGPT, like AnthropicAI's Claude model (a competitor of ChatGPT): https://twitter.com/goodside/status/1611556749726605312
+
+
+Edits vs completion endpoints:
+  - although they are based on the same Davinci models, the completion endpoint was better
+  - however, edits is still in beta, and might the preferred endpoint in the future (however, the best one would be the chatbot interface when available)
+  - Curie models were clearly inferior, but this could also be explained by the complexity of our prompts; maybe simplier prompts would work better.
+  - We show that the Davinci models are capable of revising a complex manuscript, and that any potential issue could be more related to the complexity of the prompts than to the models themselves (prompt design).
+
+
+- the models offers a wide variety of parameters to tune, for instance, "logit_bias" could be helpful to improve completion in the methods section (equations), which allows to provide a bias for specific tokens (maybe if we add "$$", the model would deal better with equations).
+- other models in the future will certainly improve the AI-assisted revision of scientific manuscripts
```

Comment:
This and the sentence below seem good to use, potentially combined into one paragraph about how things can be optimized w/ the portions above.

It might be interesting to point out the bioRxiv XML / PMC OA corpus thing that @danich1 did linking preprint/published papers -> maybe that's one way to fine tune... what would this paragraph look like in the peer reviewed paper?

Reply:
I didn't have time to include this, sorry! Definitely worth considering in the next round of review of this section.